src/panels/lovelace/hui-view.ts

```diff
@@ -233,28 +292,14 @@ class HUIView extends localizeMixin(EventsMixin(PolymerElement)) {
     this._cards = elements;
 
     if ("theme" in config) {
-      applyThemesOnElement(root, this.hass.themes, config.theme);
+      applyThemesOnElement(root, this.hass!.themes, config.theme);
     }
   }
+}
 
-  _hassChanged(hass) {
-    this._badges.forEach((badge) => {
-      const { element, entityId } = badge;
-      element.setProperties({
-        hass,
-        state: hass.states[entityId],
-      });
-    });
-    this._cards.forEach((element) => {
-      element.hass = hass;
-    });
-  }
-
-  _lovelaceChanged(lovelace) {
-    if (lovelace.editMode && !editCodeLoaded) {
-      editCodeLoaded = true;
-      import(/* webpackChunkName: "hui-view-editable" */ "./hui-view-editable");
-    }
+declare global {
+  interface HTMLElementTagNameMap {
+    "hui-view": HUIView;
   }
 }
 
```

Comment:
```suggestion
customElements.define("hui-view", HUIView);
```

Reply:
It was not in the diff because it didn't change 👍